src/config/rendering.ts

```diff
@@ -13,6 +13,11 @@ type NetworkConditions = {
   latency: number;
 };
 
+export interface TracesConfig {
+  url: string;
+  serviceName: string;
```

Comment:
```suggestion
  serviceName?: string;
```
I'd put it as optional to not have to fill them everywhere, especially in the envConfig object.